Dockerfile

```diff
@@ -20,25 +20,22 @@ WORKDIR /opt/app-root/src
 COPY ./.gemrc.prod /etc/gemrc
 COPY ./Gemfile.lock ./Gemfile /opt/app-root/src/
 
-RUN FULL_RHEL=$(microdnf repolist --enabled | grep rhel-8);                                   \
```

Comment:
@MichaelMraka I never understood the purpose of the `FULL_RHEL` variable here. I think Marley copied the snippet from your Dockerfile without context. Is there any case when the build is running on a full RHEL machine?

Reply:
Yes, when the image is built on internal jenkins.
Anyway we've already stop using it and I think you can remove it as well.